tests/github-action-helper.sh

```diff
@@ -2,31 +2,70 @@
 
 set -eEuo pipefail
 
+: "${FUNCTION:=${1}}"
+
+# source https://krew.sigs.k8s.io/docs/user-guide/setup/install/
+function install_krew() {
+  set -x; cd "$(mktemp -d)" &&
+  OS="$(uname | tr '[:upper:]' '[:lower:]')" &&
+  ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')" &&
+  curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.tar.gz" &&
+  tar zxvf krew.tar.gz &&
+  KREW=./krew-"${OS}_${ARCH}" &&
+  "$KREW" install krew
+  # We want this to output $PATH without expansion
+  # shellcheck disable=SC2016
+  echo 'export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"' >>  /home/runner/.bashrc
+  # shellcheck disable=SC1091
+  source /home/runner/.bashrc
```

Comment:
```console
runner@fv-az173-495:~/work/kubectl-rook-ceph/kubectl-rook-ceph$ kubectl krew
Error: unknown command "krew" for "kubectl"
Run 'kubectl --help' for usage.
runner@fv-az173-495:~/work/kubectl-rook-ceph/kubectl-rook-ceph$ source /home/runner/.bashrc 
runner@fv-az173-495:~/work/kubectl-rook-ceph/kubectl-rook-ceph$ kubectl krew
krew is the kubectl plugin manager.
You can invoke krew through kubectl: "kubectl krew [command]..."

Usage:
---
```

Reply:
Did you reload your shell after applying the changes to `.bashrc`?

Reply:
```suggestion
  source $HOME/.bashrc
```

Reply:
> Did you reload your shell after applying the changes to `.bashrc`?

yes, I did reload using `source ...` command

Reply:
@leseb @travisn as I'm getting an error due to shell being not reloaded but when doing manually in tmate session it is working. I was thinking to just comment out the CI changes and merge after that I'll go ahead and create PR on krew-index repo. So, that it will minimize the steps in CI and doc also. thoughts?